src/python/nimbusml/tests/preprocessing/missing_values/test_data_with_missing.py

```diff
@@ -98,7 +98,8 @@ def test_input_conversion_to_float(self):
         assert_equal(result.loc[2, 'f5'], True)
         result.loc[2, 'f5'] = False
         result = ~result
-        self.assertTrue(result.all(axis=None))
+        for val in result.all().tolist():
+            self.assertTrue(val)
 
```

Comment:
Doesn't `result.all()` return a single value (boolean) here?

Reply:
I believe not. When I obtained outputs for testing, I saw the actual values of `Result`. 

`Result `is this Series:

>      f0    f1    f2    f3    f4    f5
> 0  True  True  True  True  True  True
> 1  True  True  True  True  True  True
> 2  True  True  True  True  True  True
> 3  True  True  True  True  True  True

`Result.all()` is:

> f0    True
> f1    True
> f2    True
> f3    True
> f4    True
> f5    True

`Result.all().tolist()` is:

> [True, True, True, True, True, True]

I believe `Result.all()` returns a single value that represents a single column. It returns true iff all the values in a given column (f0, f1, ...) are true. Here's more information just in case (Pandas 0.23.0 API): https://pandas.pydata.org/pandas-docs/version/0.23.0/generated/pandas.DataFrame.all.html#pandas.DataFrame.all